src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    background-position: center;
+
+    background-image: url(../images/star.svg);
+  }
+
+  &--4 :nth-child(-n + 4) {
```

Comment:
According to the BEM methodology and the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the modifier should not be used alone. It should be used with the block or element it modifies. In this case, ensure that the `stars--4` modifier is used alongside the `stars__star` element it is intended to modify.